transmart-copy/src/main/groovy/org/transmartproject/copy/Database.groovy

```diff
@@ -91,11 +91,18 @@ class Database implements AutoCloseable {
         if (!password) {
             throw new IllegalArgumentException('Please set the PGPASSWORD environment variable.')
         }
+        Integer maximumPoolSize
+        try{
+            maximumPoolSize = Integer.parseInt(params['MAXPOOLSIZE'])
+        } catch (NumberFormatException e) {
+            maximumPoolSize = 8
```

Comment:
I don't see the need for this parameter, what is the use case?
The default value in Hikari seems to be [10](https://github.com/brettwooldridge/HikariCP#configuration-knobs-baby), is there a reason to have 8 here?
About the code: I would expect something like
```groovy
if (params['MAXPOOLSIZE']) {
  config.maximumPoolSize = Integer.parseInt(params['MAXPOOLSIZE'])
}
```
i.e., only setting it when a parameter is provided, using the default of Hikari otherwise.
Also, I would expect an error to be thrown when it can't parse instead of silently using a default instead.

Reply:
I added it because with the default max pool size I was getting this error, when running tests:
`org.postgresql.util.PSQLException: FATAL: sorry, too many clients already`
I will change it the way you mentioned.